src/features/dataset/DatasetRoutes.tsx

```diff
@@ -26,6 +28,12 @@ const DatasetRoutes: React.FC<{}> = () => {
   // ref. Move all ref constructino into container components to avoid potential
   // bugs
   const qriRef = newQriRef(useParams())
+  const dispatch = useDispatch()
+
+  useEffect(() => {
+    dispatch(loadDatasetVersionInfo(qriRef))
+    //eslint-disable-next-line
```

Comment:
is the line below complaining about an empty array? it'd be better to remove the array entirely and avoid the eslint disable.